src/assets/country-information/country-information.jsx

```diff
@@ -0,0 +1,68 @@
+import axios from "axios";
+import {useState} from "react";
+import './country-information.css'
+import roundToMillions from "../../helpers/roundToMillions.js";
+
+function CountryInformation() {
+    const [data, setData] = useState(null)
+    const [inputValue, setInputValue] = useState('')
+    const [error, setError] = useState('')
+
+   const handleEnter = (e) => {
+        if (e.keyCode === 13) {
```

Comment:
`keyCode` is deprecated: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode 😉